modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-labels-and-tags_{context}"]
+= Configuring the user-defined labels and tags for GCP
+
+
+Google Cloud Platform (GCP) provides `labels` and `tags` that help to identify and organize the resources created for a specific {product-title} cluster, making them easier to manage. 
+
+You can define `labels` and `tags` for each GCP resource only during {product-title} cluster installation. 
+
+[IMPORTANT]
+====
+User-defined `labels` and `tags` are not supported for {product-title} clusters upgraded to {product-title} 4.14 version.
+====
+
+.User-defined labels
+
+User-defined `labels` and {product-title} specific labels are applied only to resources created by {product-title} installer and its core operators such as: 
+* `machine-api-provider-gcp` 
+* `cluster-image-registry-operator`
+* `gcp-filestore-csi-driver-operator`
+* `gcp-pd-csi-driver-operator` 
+
+The user-defined `labels` and {product-title} specific labels are not applied on the resources created by any other operators or the kubernetes in-tree components that create resources, for example, the Ingress load balancers.
+
+The following are the requirements for the user-defined labels:
```

Comment:
This should be present in a location when user is trying to create a label.

Reply:
I will re-arrange the info while creating the seperate modules.

Reply:
Not addressed